src/Mod/Arch/ArchFloor.py

```diff
@@ -121,17 +165,36 @@ def Activated(self):
 
 
 class _Floor(ArchIFC.IfcProduct):
+    """The Floor object.
```

Comment:
The documentation of this ArchFloor file is good, however, this file is essentially obsolete. In the past, a Floor was this `ArchFloor._Floor` class, which was basically inherited from a Std_Group, `App::DocumentObjectGroup`.

However, later it was remade into a BuildingPart. You can find the definition in `ArchBuildingPart.py`. It basically creates a BuildingPart and then sets `obj.IfcType = "Building Storey"`.

Reply:
Resolved with 96062160a23e939e2c4028cd1bbcf8a781a559d7